frontend/src/components/Hexagon.css

```diff
@@ -22,3 +32,18 @@
 .hexagon .W {
   fill: antiquewhite;
 }
+
+@keyframes pulse {
+  0% {
+    fill:rgb(113, 242, 214);
+  }
+
+  50% {
+    fill:rgb(146, 255, 231);
+  }
+
+
```

Comment:
a lot of empty lines; would also encourage everything to be ordered alphabetically

Reply:
not sure alphabetical is the most intuitive way? But there should be some kind of ordering